tests/converter/test_converter.py

```diff
@@ -0,0 +1,38 @@
+import os
```

Comment:
we should rename this file to `test_df_converter`

Reply:
done